core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -262,13 +264,6 @@ class TimeSeriesShard(val dataset: Dataset,
   private[core] val overflowBlockFactory = new BlockMemFactory(blockStore, None, dataset.blockMetaSize, true)
   val partitionMaker = new DemandPagedChunkStore(this, blockStore, chunkRetentionHours)
 
-  /**
-    * Unencoded/unoptimized ingested data is stored in buffers that are allocated from this off-heap pool
-    * Note that this pool is also used to store partition keys.
-    */
-  logger.info(s"Allocating $bufferMemorySize bytes for WriteBufferPool/PartitionKeys for " +
```

Comment:
Can you please add the logging back where you are declaring the new nativeBufferManager?